### PR TITLE
D16b geometry

### PR DESCRIPTION
**Pull request by MathieuTillet**
The geometry for D16 new detector

### CODE REVIEW

File: ILL/IDF/d16b_generateIDF.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+from helper import MantidGeom
+
+# unit is metre
+instrumentName = 'D16B'
+validFrom = "2022-12-01 00:00:00"
+
+monochromator_source = -2.8
+
+# 2 monitors
+zMon1 = -1  # TODO distance ?
```
Comment (gui-co) [inline ILL/IDF/d16b_generateIDF.py:18]: Can we resolve the TODO ? I do not know if the precise distance is useful here, if not, we can keep -1.

**Review by gui-co — COMMENTED** [review]
This looks fine. There is just a remaining TODO which could be removed.

File: ILL/IDF/d16b_generateIDF.py (hunk 1 above)
Comment (darominski) [reply ILL/IDF/d16b_generateIDF.py:18]: The Time of flight is not relevant in this case, so an arbitrary distance should be fine. I am going to update the comment.

**Review by gui-co — APPROVED** [review]
Thanks for the modification.

**Pull request merged**